### PR TITLE
String Check.py

### PR DESCRIPTION
**Pull request by aniketprasadgit**
1. Insert the given string in a set
2. Check if the set characters consist of 1 and/or 0 only.

### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cozek** [comment]
you can also check with ( perhaps within a `try` and `except`)
```python
int(some_string,2)
```
Also, tests are failing. Please read the contributions guidelines and rectify.

File: String Check.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Python program to check 
+# if a string is binary or not 
+
+# function for checking the 
+# string is accepted or not 
+def check(string) :
```
Comment (mateuszz0000) [inline String Check.py:6]: Better to return `bool` instead print True/False

File: String Check.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Python program to check 
+# if a string is binary or not 
+
+# function for checking the 
+# string is accepted or not 
+def check(string) : 
+
+	# set function convert string
```
Comment (mateuszz0000) [inline String Check.py:8]: PLease add doctests and typehints

**Review by mateuszz0000 — CHANGES_REQUESTED** [review]

File: String Check.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	# declare set of '0', '1' . 
+	s = {'0', '1'} 
+
+	# check set p is same as set s 
+	# or set p contains only '0' 
+	# or set p contains only '1' 
+	# or not, if any one condition 
+	# is true then string is accepted 
+	# otherwise not . 
+	if s == p or p == {'0'} or p == {'1'}:
```
Comment (mateuszz0000) [inline String Check.py:21]:
```
try:
    int(string, 2)
    return True
except:
   return False
```

**Review by mateuszz0000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**